tracing-log/tests/log_tracer.rs

```diff
@@ -56,6 +56,12 @@ impl Subscriber for TestSubscriber {
 fn normalized_metadata() {
     LogTracer::init().unwrap();
 
+    let my_file = if cfg!(windows) {
```

Comment:
What do you think about just using the `file!()` macro here instead? That way, the value of `my_file` will always reflect the actual file name, even if we were to rename the test file or move it, and will handle Windows compatibility without having to do so manually.